tests/runtime/engine/runner.py

```diff
@@ -284,6 +285,15 @@ def run_test(test):
             if after and after.poll() is None:
                 os.killpg(os.getpgid(after.pid), signal.SIGKILL)
 
+        if test.cleanup:
+            try:
+                cleanup = subprocess.run(test.cleanup.split(), shell=True, capture_output=True, text=True)
```

Comment:
This is not correct after all, you should drop the `.split()` call. [subprocess.Popen documentation](https://docs.python.org/3/library/subprocess.html#subprocess.Popen) says:
> The shell argument (which defaults to False) specifies whether to use the shell as the program to execute. If shell is True, it is recommended to pass args as a string rather than as a sequence.